src/org/zaproxy/zap/extension/frontendscanner/files/scripts/templates/client-side-passive/scan.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Client-side scripts are run as functions on the target domain.
+ * Thus, they should be written in Javascript.
+ *
+ * They receive `frontEndScanner` as a parameter, which exposes:
+ *   - A `mailbox` attribute to subscribe to events happening in the page
+ *   (more about that: https://github.com/zaproxy/front-end-tracker).
+ *   - A `zapAlertConstants` attribute, with useful data to create alerts.
+ *   - A `reportAlertToZap` function to create an Alert in ZAP.
+*/
+
+// Use the mailbox to react to any storage interaction with a lambda function
+// which call our `report()` one every time the storage is set.
```

Comment:
This was not changed.

Reply:
On purpose.
`one` refers to the word `function` earlier in the sentence.

Reply:
Oh, OK, was not reading it that way :)

Reply:
Should it be "once"?

Reply:
:cry: 

Was meant to avoid the repetition of the word "function" in:

> Use the mailbox to react to any storage interaction with a lambda function which call our `report()` **function** every time the storage is set.

As it's easily misinterpretable, gonna change it.

Reply:
LGTM!

Reply:
Slightly changed the phrase: less words to say the same thing.

P.S: I prefer to keep `method` when talking about OOP.
Javascript has functions (and prototypal inheritance), and no classes (just objects).